services/api/src/clients/pubSub.ts

```diff
@@ -1,32 +1,40 @@
 import { path } from 'ramda';
 import { withFilter } from 'graphql-subscriptions';
-import { AmqpPubSub } from 'graphql-rabbitmq-subscriptions';
+import { RedisPubSub } from 'graphql-redis-subscriptions';
 import { ForbiddenError } from 'apollo-server-express';
 import { logger } from '../loggers/logger';
-import { getConfigFromEnv } from '../util/config';
 import { query } from '../util/db';
 import { Sql as environmentSql } from '../resources/environment/sql';
 import { ResolverFn } from '../resources';
+import { config } from './redisClient';
+import Redis from 'ioredis';
```

Comment:
`ioredis` is the older library, and while it's technically still maintained, it's suggested to use `node-redis` for all new work. The docs for the subscription library default to ioredis but it can also support node-redis, which is what we are using already in the [redisClient.ts](https://github.com/uselagoon/lagoon/blob/main/services/api/src/clients/redisClient.ts).

The `publisher`/`subscriber` need independent objects, so a small refactor of redisClient to export a `createRedisClient` method OR just copy/paste it.

Reply:
Feel free to fix it to how you think it should be otherwise